cmd/jujud/agent/model/manifolds_test.go

```diff
@@ -4,22 +4,28 @@
 package model_test
 
 import (
-	"github.com/juju/testing"
 	jc "github.com/juju/testing/checkers"
 	"github.com/juju/utils/clock"
 	"github.com/juju/utils/set"
 	gc "gopkg.in/check.v1"
 
 	"github.com/juju/juju/cmd/jujud/agent/model"
+	"github.com/juju/juju/feature"
+	"github.com/juju/juju/testing"
 	"github.com/juju/juju/worker/workertest"
 )
 
 type ManifoldsSuite struct {
-	testing.IsolationSuite
+	testing.BaseSuite
 }
 
 var _ = gc.Suite(&ManifoldsSuite{})
 
+func (s *ManifoldsSuite) SetUpTest(c *gc.C) {
+	s.SetInitialFeatureFlags(feature.CrossModelRelations)
```

Comment:
can you please do this in a separate test? i.e. check the standard set of manifolds, and then have another test which checks what the effect of setting the feature flag is